docs/FAQ.md

```diff
@@ -8,6 +8,10 @@ The HashiCorp Terraform vSphere provider team is :
 * Alex Pilon, Engineer - [@appilon](https://github.com/appilon)
 * Brandy Jackson, Engineering Manager - [@ibrandyjackson](https://github.com/ibrandyjackson)
 
+Collaborators from VMware are :
+
+* Ryan Johnson - [@tenthirtyam](https://github.com/tenthirtyam)
```

Comment:
We can include your title if you prefer? Just let us know:
```
* Ryan Johnson, Staff II Solutions Architect - [@tenthirtyam](https://github.com/tenthirtyam)
```

Reply:
Yes, please - thank you!